tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -39,26 +39,23 @@ import {isDatumVisible} from './utils';
 
 @Component({
   selector: 'scalar-card-data-table',
-  template: `
-    <tb-data-table
-      [headers]="columnHeaders"
-      [data]="getTimeSelectionTableData()"
-      [sortingInfo]="sortingInfo"
-      [columnCustomizationEnabled]="columnCustomizationEnabled"
-      [smoothingEnabled]="smoothingEnabled"
-      (sortDataBy)="sortDataBy.emit($event)"
-      (orderColumns)="orderColumns($event)"
-    >
-      <ng-container header>
-        <ng-container *ngFor="let header of columnHeaders">
-          <tb-data-table-header-cell
-            *ngIf="header.enabled"
-            [header]="header"
-            [sortingInfo]="sortingInfo"
-          ></tb-data-table-header-cell> </ng-container
-      ></ng-container>
-    </tb-data-table>
-  `,
+  templateUrl: 'scalar_card_data_table.ng.html',
+  styles: [
+    `
+      .row-circle {
+        height: 12px;
+        width: 12px;
+      }
+      .row-circle > span {
+        border-radius: 50%;
+        border: 1px solid rgba(255, 255, 255, 0.4);
```

Comment:
How does this look in dark mode?

Reply:
This style is copied so it looks the same as before. I checked it and I think it looks just fine in dark mode.